api/proof/routes.go

```diff
@@ -0,0 +1,179 @@
+package proof
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/ElrondNetwork/elrond-go/api/errors"
+	"github.com/ElrondNetwork/elrond-go/api/middleware"
+	"github.com/ElrondNetwork/elrond-go/api/shared"
+	"github.com/ElrondNetwork/elrond-go/api/wrapper"
+	"github.com/gin-gonic/gin"
+)
+
+const (
+	getProofEndpoint    = "/proof/root-hash/:roothash/address/:address"
+	verifyProofEndpoint = "/proof/verify"
+	getProofPath        = "/root-hash/:roothash/address/:address"
```

Comment:
are `getProofPath` and `verifyProofPath` used?

Reply:
Yes, they are used in `Routes()` func. That is the pattern used by all the other api subpackages.